### PR TITLE
Issue #579, apply calibration 

### LINKED ISSUE #579: Apply sensor calibration
When extracting data from a file, perform the necessary sensor calibration on it.

Look in `calibration` table for matching `instrument_id` and `target = sensorName`, and `type = SENSOR_CALIBRATION`. Instantiate the class and apply the formula.

The calibration should probably be applied somewhere around line 824 in DataSetRawData.java in package uk.ac.exeter.QuinCe.data.Dataset

### LINKED ISSUE #768: Script db_restore_from_prod not removing nonexistent files from filestore
The db_restore_from_prod - script should restore a pristine database and filestore. Currently, the options give to rsync for syncing the file store, does not remove files added locally. This will lead to inconsistencies in the QuinCe data.

Possible fix: Add `--delete` to the rsync options to make sure files are removed from the file store.

To test: 
- add a file to your local filestore
- run ./scripts/db_restore_from_prod.sh
- Verify only files in the filestore on poseidon now exists in your local file store

### LINKED ISSUE #776: new_dataset: Trying to disable button before DOM is ready
In new_dataset.xhtml a script is trying to disable a button before the DOM is ready, leading to the following js-errors in the console: 

Widget for var 'pAddButton' not available!
dataset_list.xhtml:13 Uncaught TypeError: Cannot read property 'disable' of undefined
    at dataset_list.xhtml:13

The issue can be fixed by wrapping the `PF('pAddButton').disable();` - statement around line 30 like this: 

```
$( document ).ready(function() {
  // Make sure DOM is loaded befor calling this
  PF('pAddButton').disable();
});

```

To test: 
- Log in to quince
- open javascript console (ctrl-shift-c)
- click on Create new data set - button, and verify the errors in the js console have disappeared

### PR DESCRIPTION
**Pull request by jonasfh**
This patch applies calibration to the data extraction of values needing polynomial calibration, like mV - values of temperature etc.

The pull request also includes a small fix for resetting the database from prod, and removing a js error in new_dataset.xhtml

Closing issues: 
fix #579
fix #776 
fix #768

### CODE REVIEW

**Comment by jonasfh** [comment]
There is an issue if you try to define a dataset, but it don't work because you have not added calibrations yet. If you then go to the calibrations, and add calibrations, thereafter return to add the dataset, it will still not work. This has to do with the current session-scoping of the dataset-bean. I could handled this by re-initializing the bean, but I think we should rather make this page view-scoped, it makes more sense that way.

**Comment by squaregoldfish** [comment]
Yes, scoping of pages is something that needs looking at in general (see #500).

**Comment by squaregoldfish** [comment]
What's the workaround? Do you have to switch instruments on the Data Sets list page?

**Comment by jonasfh** [comment]
Switching instrument might reload it. Otherwise you need to log out and log back in.

File: WebApp/src/uk/ac/exeter/QuinCe/data/Instrument/Calibration/CalibrationDB.java [hunk 1]
```diff
@@ -112,7 +116,8 @@ public CalibrationSet getCurrentCalibrations(DataSource dataSource, long instrum
       stmt = conn.prepareStatement(GET_RECENT_CALIBRATIONS_QUERY);
       stmt.setLong(1, instrumentId);
       stmt.setString(2, getCalibrationType());
-
+      // Get epoc milliseconds
+      stmt.setLong(3, date.toEpochSecond(ZoneOffset.UTC) * 1000);
```
Comment (squaregoldfish) [inline WebApp/src/uk/ac/exeter/QuinCe/data/Instrument/Calibration/CalibrationDB.java]: Use `DateTimeUtils.dateToLong`

Comment (jonasfh) [reply WebApp/src/uk/ac/exeter/QuinCe/data/Instrument/Calibration/CalibrationDB.java]: Changed this!

**Pull request merged**